test/system/commander/index.js

```diff
@@ -0,0 +1,43 @@
+const program = require('commander');
```

Comment:
any particular reason this file gets its own directory? Could it be moved back to test/system and be called commander.js ?